exercises/difference-of-squares/src/test/java/DifferenceOfSquaresCalculatorTest.java

```diff
@@ -4,10 +4,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-/*
- * version: 1.1.0
- */
-public final class DifferenceOfSquaresCalculatorTest {
+public class DifferenceOfSquaresCalculatorTest {
```

Comment:
Looks like we lost the version comment here; could you please restore it? Thanks!